bundles/org.openhab.ui/web/src/components/widgets/plan/oh-plan-page.vue

```diff
@@ -101,6 +102,12 @@ export default {
           type: 'INTEGER',
           description: 'The width of the image (by default 1000 pixels). Please specify if the image is not square to compute the aspect ratio'
         },
+        {
+          name: 'noZoomOrDrag',
+          label: 'Disable Ability to Zoom, Drag or Move',
```

Comment:
Isn't dragging and moving pretty much the same thing in this context?

```suggestion
          label: 'Disable Zooming & Dragging',
```